packages/apps/reputation-oracle/server/src/modules/kyc/kyc.service.spec.ts

```diff
@@ -218,46 +186,73 @@ describe('Kyc Service', () => {
     const mockKycUpdate = {
```

Comment:
Would be nice to not change the properties in `mockKycUpdate` between tests, but generate a new value with expected values before each test